src/main/java/blink/command/Command.java

```diff
@@ -0,0 +1,27 @@
+package blink.command;
+
+import blink.Storage;
+import blink.TaskList;
+import blink.Ui;
+
+/**
+ *  Command object that is created when a recognized command is input.
+ */
+abstract public class Command {
```

Comment:
Quite a cool idea in making the Command class an abstract class! I think not many people do it 👍